pykern/pkcli/fmt.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+u"""Wrapper for Python formatter (currently, ``black``) to update and to validate a repository.
+
+:copyright: Copyright (c) 2015 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkdebug import pkdp
+import pykern.pksubprocess
+import pykern.pkunit
+
+def run(path):
+    '''
+        Run black formatter on path
+
+        Args:
+            path - string path to file or dir
+    '''
+    pykern.pksubprocess.check_call_with_signals(['black', f'{path}'])
+
+
+def diff(path):
+    pykern.pksubprocess.check_call_with_signals([
+        'black',
+        '--diff',
```

Comment:
Add `--quiet` which does the right thing (shows the diff, but not the emojis or statements like "unchanged")

diff should return false or true if there are differences. This  can be accomplished by `black --check --diff`.